### PR TITLE
feat: add slugs and email notes post-ingestions

### PR DESCRIPTION
**Pull request by Ali-D-Akbar**
[PROD-3287](https://2u-internal.atlassian.net/browse/PROD-3287)
Ingestion Email updates.

### CODE REVIEW

File: course_discovery/apps/course_metadata/templates/course_metadata/email/loader_ingestion.html [hunk 1]
```diff
@@ -83,6 +83,6 @@ <h3>Ingestion Failures</h3>
 {% endif %}
 
 <p>
-    <strong>Note: For queries or comments, reach out to {{ingestion_contact_email}} via email.</strong>
+    <strong>Note: Products are ingested in Catalog and currently not published. They will be live on http://edx.org in next 24 hours once Prospectus build runs. For queries or comments, reach out to {{ingestion_contact_email}} via email.</strong>
```
Comment (DawoudSheraz) [inline course_discovery/apps/course_metadata/templates/course_metadata/email/loader_ingestion.html]: Prospectus is internal, so I would suggest that this is made part of a setting variable and added here via context variable.

File: course_discovery/apps/course_metadata/templates/course_metadata/email/loader_ingestion.html [hunk 2]
```diff
@@ -49,7 +49,7 @@ <h3>New Products</h3>
         <ul>
             {% for new_product in created_products %}
                 {% if product_type != "DEGREES" %}
-                    <li><a href="{{publisher_url}}courses/{{new_product.uuid}}">{{ new_product.uuid }}</a>{% if new_product.external_course_marketing_type %} ({{ new_product.external_course_marketing_type }}) {% endif %}</li>
+                    <li><a href="{{publisher_url}}courses/{{new_product.uuid}}">{{ new_product.uuid }}</a><pre> - </pre>{{ new_product.url_slug}} {% if new_product.external_course_marketing_type %} ({{ new_product.external_course_marketing_type }}) {% endif %}</li>
                 {% else %}
                     <li>{{ new_product.uuid }}</li>
```
Comment (DawoudSheraz) [inline course_discovery/apps/course_metadata/templates/course_metadata/email/loader_ingestion.html:54]: I would suggest that add slugs for Degrees as well.

File: course_discovery/apps/course_metadata/templates/course_metadata/email/loader_ingestion.html (hunk 1 above)
Comment (Ali-D-Akbar) [reply course_discovery/apps/course_metadata/templates/course_metadata/email/loader_ingestion.html]: Addressed

Comment (DawoudSheraz) [reply course_discovery/apps/course_metadata/templates/course_metadata/email/loader_ingestion.html]: Well, I should have been more explicit. That particular statement that changes will be live in 24 hours is edx internal specific. We can make it more generic, like once the build for <service> runs, the changes will be live on <site>. The avg turnaround time is 24-48 hours.

Comment (Ali-D-Akbar) [reply course_discovery/apps/course_metadata/templates/course_metadata/email/loader_ingestion.html]: Added.

File: course_discovery/apps/course_metadata/templates/course_metadata/email/loader_ingestion.html [hunk 3]
```diff
@@ -83,6 +83,9 @@ <h3>Ingestion Failures</h3>
 {% endif %}
 
 <p>
-    <strong>Note: For queries or comments, reach out to {{ingestion_contact_email}} via email.</strong>
+    <strong>Note: Products are ingested in Catalog and currently not published.
+        The changes will be live on http://edx.org once {{marketing_service_name}} build runs.
```
Comment (DawoudSheraz) [inline course_discovery/apps/course_metadata/templates/course_metadata/email/loader_ingestion.html:87]: nit: there will be a setting var for edx.org in Discovery. use that

**Review by DawoudSheraz — APPROVED** [review]

**Review by AfaqShuaib09 — APPROVED** [review]

**Pull request merged**